README.md

```diff
@@ -63,7 +63,7 @@ The target enables developers to easily execute medium-size Language Models (LLM
 > Spezi LLM Local is not compatible with simulators. The underlying [`mlx-swift`](https://github.com/ml-explore/mlx-swift) requires a modern Metal MTLGPUFamily and the simulator does not provide that.
 
 > [!IMPORTANT]
-> Important: To use the LLM local target, some LLMs require adding the [Increase Memory Limit](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_kernel_increased-memory-limit) entitlement to the project.
+> Important: To use the LLM local target, some LLMs require adding the *Increase Memory Limit* entitlement to the project.
```

Comment:
We should not refer to entitlements in the docs that may don't even exist anymore in up-to-date SDKs (as indicated by the non-existant docs)? 
Maybe @LeonNissen has more input on the state of these entitlements?

Reply:
Discussed this with @LeonNissen. The entitlements still exist and are necessary, it's just the pages on Apple's website that are missing for some reason. These entitlements are also mentioned in other parts of Apple's website that still link to these pages. We'll remove the links for now and replace them if the pages come back online.